packages/std/src/imports.rs

```diff
@@ -18,7 +18,7 @@ use crate::{
 };
 
 /// An upper bound for typical canonical address lengths (e.g. 20 in Cosmos SDK/Ethereum or 32 in Nano/Substrate)
-const CANONICAL_ADDRESS_BUFFER_LENGTH: usize = 32;
+const CANONICAL_ADDRESS_BUFFER_LENGTH: usize = 54;
```

Comment:
I think this makes `54` the effective max, right? Is that okay?

Reply:
Right. I think it would be better to use something generic, like 64. The chain can then decide on any length up to 64 bytes.

Reply:
Should we just change it to 64

Reply:
prob good practice to do it in another PR im guessing

Reply:
Yes. Let's do it here, because the change 32 -> 54 does not make much sense.